tests/phpunit/bootstrap.php

```diff
@@ -11,8 +11,6 @@
 	xdebug_disable();
 }
 
-require_once TESTS_PLUGIN_DIR . '/vendor/autoload.php';
```

Comment:
This is not needed depending on how you're using PHPUnit in this project. Let's wrap it in a `file_exists()` check as you suggested, in order to account for either case.

Reply:
Sure, sounds good. The tests can be run directly by placing the plugin in `/src/wp-content/plugins` then starting phpunit from `/src/wp-content/plugins/wp-lazy-loading`.